lmdeploy/cli/entrypoint.py

```diff
@@ -11,13 +10,8 @@
 def run():
     """The entry point of running LMDeploy CLI."""
     args = sys.argv[1:]
-    is_deprecated_chat_cli = len(args) > 1 and args[0] == 'chat' and \
-        args[1] in ['torch', 'turbomind']
     CLI.add_parsers()
-    if is_deprecated_chat_cli:
-        SubCliChat.add_parsers()
-    else:
-        CLI.add_parser_chat()
+    CLI.add_parser_chat()
```

Comment:
could move this to CLI.add_parsers() 
https://github.com/InternLM/lmdeploy/blob/c9c225fb3e1aa2cbfa654ec4a49b49d9b2a4fbbd/lmdeploy/cli/cli.py#L276-L280